test/Error/rlexe.xml

```diff
@@ -30,6 +30,8 @@
     <default>
       <files>stack.js</files>
       <baseline>stack.baseline</baseline>
+      <!-- xplat-todo: Understand why this test is failing on OS X -->
+      <tags>exclude_mac</tags>
```

Comment:
Hmm, interesting- it had failed in my test PR which is why I had disabled it, but I just reran the tests on that PR and that test passed. However, another test failed now, which worries me. @obastemur - are the tests on OS X stable, or do we need to temporarily disable tests for that platform until we improve the stability? See #1329 for logs for the current failure. I'm not on corpnet right now, I'll try to investigate more once I'm in the office.